python/cugraph/cugraph/cores/CMakeLists.txt

```diff
@@ -0,0 +1,32 @@
+# =============================================================================
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+# =============================================================================
+
+set(cython_sources k_core_wrapper.pyx)
+set(linked_libraries cugraph::cugraph)
+rapids_cython_create_modules(
+  CXX
+  SOURCE_FILES "${cython_sources}"
+  LINKED_LIBRARIES "${linked_libraries}" MODULE_PREFIX cores_
+)
+
+set(targets_using_numpy cores_k_core_wrapper)
+foreach(target IN LISTS targets_using_numpy)
+  target_include_directories(${target} PRIVATE "${Python_NumPy_INCLUDE_DIRS}")
+endforeach()
+
+if(NOT cugraph_FOUND)
```

Comment:
Why is this block in all the CMakeLists.txt files conditional on the C++ library being found?

Reply:
Addressed